.vscode/extensions.json

```diff
@@ -1,7 +1,3 @@
 {
-	// See https://go.microsoft.com/fwlink/?LinkId=827846
-	// for the documentation about the extensions.json format
-	"recommendations": [
-		"Vue.volar"
-	]
-}
\ No newline at end of file
+  "recommendations": ["esbenp.prettier-vscode", "Vue.volar"]
```

Comment:
We had “Vue language support (Volar)” in the recommended extensions, but not Prettier.

I also looked at whether we need the [“TypeScript Vue Plugin (Volar)”](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin) extension. Is it confusing that it's also called “Volar” in parenthesis? You bet. Is it hard to use because you need to go and disable the built-in TypeScript extension? Yup. Is it unclear what it does and why you might need it? Completely.

Turns out if you read 2 different pages of the Vue docs:

- [Tooling > IDE Support](https://vuejs.org/guide/scaling-up/tooling.html#ide-support)
- [Using Vue with TypeScript > IDE Support](https://vuejs.org/guide/typescript/overview.html#ide-support)

Then the first one tells you to use the “Vue Language Features (Volar)” extension, and the second one tells you that using that other extension is for making `.ts` files understand `.vue` imports.

I tried with and without the second extension and its usefulness seems limited, so I’d rather not recommend it to contributors, given how it’s _super confusing_ and also it’s hard to configure.

Reply:
Ah yes, I remember being similarly confused but didn't dig that deep. Yeah, I agree with your recommendation here.